.github/workflows/publish-commit-bottles.yml

```diff
@@ -338,8 +384,7 @@ jobs:
             exit 1
           fi
 
-          gh pr merge --auto --merge '${{inputs.pull_request}}'
-          gh pr review --approve '${{inputs.pull_request}}'
+          gh pr merge --auto --merge "$PR"
 
       - name: Post comment on failure
         if: >
```

Comment:
If you don't mind getting spammed by `gh pr review --approve` failures -- let's post a comment for that too? 😄

Reply:
I do mind; that's why I implemented a retry! But, fine: 2d350dc2447c55aa3ff37bfdd9ab920e7dbc60d3